vertical-pod-autoscaler/deploy/admission-controller-deployment.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       serviceAccountName: vpa-admission-controller
       containers:
       - name: admission-controller
-        image: k8s.gcr.io/vpa-admission-controller:v0.1.0
+        image: k8s.gcr.io/vpa-admission-controller:0.2.0
```

Comment:
Why are we removing v from the TAG?

Reply:
This seems to be the convention for other images in the k8s.gcr.io/ repo